extensions/wrapper/wrapper-api/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractTerminationRequest.java

```diff
@@ -27,16 +29,20 @@
 @RequiredArgsConstructor
 @Schema(description = "Data for terminating a Contract Agreement")
 public class ContractTerminationRequest {
-
-    @Schema(
-        title = "Termination reason",
-        description = "A short reason why this contract was terminated",
-        requiredMode = Schema.RequiredMode.REQUIRED)
-    String reason;
+    public static final int MAX_REASON_SIZE = 100;
+    public static final int MAX_DETAIL_SIZE = 1000;
```

Comment:
why the constants, do you re-use them elsewhere?

didn't we talk about the limit of the details? this limit is too small, right?